src/app/static/src/js/runWorkflowProgress.ts

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue';
+import $ from 'jquery';
+
+import runWorkflowProgress from './components/runWorkflow/runWorkflowProgress.vue'
+
+$(document).ready(() => {
+    if ($('#runWorkflowProgressVueApp').length > 0) {
+        new Vue({
+            el: '#runWorkflowProgressVueApp',
+            components: {
+                runWorkflowProgress: runWorkflowProgress
+            }
+        });
+    }
+});
```

Comment:
Don't think you need this file - the runWorkflowTabsVueApp is the container for the progress component to. In facto I don't think there is any element with this id.